exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/ParquetGroupScan.java

```diff
@@ -864,6 +872,14 @@ public String getDigest() {
     return toString();
   }
 
+  public void setCacheFileRoot(String cacheFileRoot) {
+    this.cacheFileRoot = cacheFileRoot;
+  }
+
+  public void setBatchSize(long batchSize) {
+    this.recommendedBatchSize = batchSize;
```

Comment:
checking against current value is fine for now.  Adding new methods will be a bit outside the scope of this JIRA.